msteams-platform/toolkit/create-new-project.md

```diff
@@ -8,30 +8,30 @@ ms.topic: overview
 ms.date: 11/29/2021
 ---
 
-# Create new Teams project using Teams Toolkit
+# Create a new Teams app using Teams Toolkit
 
-You can create new Teams project using Teams Toolkit either by using **Create new project** or **Create from samples**.
+Once you have Teams Toolkit installed you have two options for creating a new app. You can either start from a basic starter project using the **Create a new Teams app** option, or you can adapt a sample app by using the **View samples** option.
```

Comment:
You can create new Teams app after installing Teams Toolkit using either one of the options - Create a new Teams app or View samples option.

Reply:
How about:

You can create new Teams app after installing Teams Toolkit using one of two options: **Create a new Teams app**, or **View samples**. This allows you the choice of creating a basic starter project, or adapting one of a number of sample apps.